exec/java-exec/src/main/java/org/apache/drill/exec/testing/ExecutionControls.java

```diff
@@ -80,7 +80,7 @@
      * @param ttl  the number of queries for which this option should be valid
      */
     public ControlsOptionValidator(final String name, final String def, final int ttl) {
-      super(name, OptionValue.Kind.STRING, OptionValue.createString(OptionType.SESSION, name, def));
+      super(name, OptionValue.Kind.STRING, OptionValue.createString(OptionType.SYSTEM, name, def));
```

Comment:
I don't think this requires a comment in code, since this applies for all option validators.